src/plugins/common/CommonTypes.hpp

```diff
@@ -53,6 +53,18 @@ struct ShadowSocksServerObject
     JSONSTRUCT_REGISTER(ShadowSocksServerObject, A(method), F(address, port, password))
 };
 
+
+//
+// Trojan Server
+struct TrojanServerObject
+{
+    QString address = "0.0.0.0";
+    QString password;
+    int port = 0;
```

Comment:
default 443?

Reply:
Yes, 443 is a better default value for trojan, thanks for mentioning.